civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -34,20 +34,20 @@
   $statusesCount[0] ++;
 endforeach;
 ?>
-<div class="chr_table-w-filters chr_table-w-filters--documents row">
+<div data-ta-documents ng-controller="myController" class="chr_table-w-filters chr_table-w-filters--documents row">
   <div class="chr_table-w-filters__filters col-md-3">
     <div class="chr_table-w-filters__filters__dropdown-wrapper">
       <div class="chr_custom-select chr_custom-select--full">
         <select class="chr_table-w-filters__filters__dropdown skip-js-custom-select">
-<?php foreach ($statuses as $key => $value): ?>
+          <?php foreach ($statuses as $key => $value): ?>
```

Comment:
@dhavalsharma `$statuses` is a php array and it's the php template file. i think we can tweak to use `ng-repeat` but i prefer not to mess with the template that is already working fine. At the end we are only attaching angular app to display modal on clicking "Open" button.